addons/io/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitAccessoryUpdater.java

```diff
@@ -47,15 +47,16 @@ public void subscribe(GenericItem item, String key, HomekitCharacteristicChangeC
         }
         ItemKey itemKey = new ItemKey(item, key);
         if (subscriptionsByName.containsKey(itemKey)) {
-            logger.error("Received duplicate subscription on {}", item.getName());
+            logger.debug("Received duplicate subscription on item {} for key {}", item.getName(), key);
         }
         subscriptionsByName.compute(itemKey, (k, v) -> {
             if (v != null) {
-                logger.error("Received duplicate subscription on {}", item.getName());
+                logger.debug("Compute: received duplicate subscription on item {} for key {}. Will unsubscribe.", item.getName(), key);
                 unsubscribe(item, key);
             }
             Subscription subscription = (changedItem, oldState, newState) -> callback.changed();
             item.addStateChangeListener(subscription);
+            logger.debug("Successfully added subscription for item '{}' using key '{}'", item.getName(), key);
```

Comment:
Without quotes like the rest of the logging messages.

```suggestion
            logger.debug("Successfully added subscription for item {} using key {}", item.getName(), key);
```